awx/main/credential_plugins/dsv.py

```diff
@@ -2,7 +2,7 @@
 
 from django.conf import settings
 from django.utils.translation import gettext_lazy as _
-from thycotic.secrets.vault import SecretsVault
+from delinea.secrets.vault import SecretsVault
```

Comment:
Also see https://github.com/ansible/awx/pull/14207, it looks like we have 2 credential plugins that import this library. And 2 multiple open PRs doing some fraction of the upgrade.

Reply:
I think we should rebase and merge the maintainers PR and then I can rebase and include that work.

Reply:
That's good with me, I asked the question in that PR, because we should strive to keep all plugins functional.

Reply:
I had additional conversation in that PR https://github.com/ansible/awx/pull/14207 and left it with a minor request. I'm completely happy to get that in on its own. Knowing that, it would seem appropriate to remove that change from this python upgrade PR.

Reply:
Errors installing them (which breaks all builds), errors when using the plugin, or more subtle hypothetical python version compatibility issues which might not affect us anyway?

The PR https://github.com/ansible/awx/pull/14207 has some clear steps forward, but they have not been responded too. Additionally, upgrading `python-dsv-sdk` is not attempted by that. Manually testing imports, I have found that upgrading `python-dsv-sdk` looks fairly straightforward and easy. In both cases, upgrading the library changes the import path.

If those changes really are needed, you could consider taking over that patch from them, including it in here, or doing a quick separate PR where you upgrade those 2 libraries. We have somewhat of a problem that we have a fragile testing situation, for which I have some potential work up at https://github.com/AlanCoding/awx/pull/81. If you want to verify functionality of these plugins after upgrade, you could try the manual instructions at `tools/docker-compose/README.md` which have been tested quite recently.